### ISSUE TITLE
External configuration webinterface

### ISSUE THREAD

**Issue by LegendaryB** [TBSniller/piccap#17]
As already pointed out on discord I would like the idea to have a external webinterface. So that you would be able to configure PicCap with a browser from a desktop or a smartphone instead of using the app on the TV.

It could be implemented like that: standalone app for example in Google Go (native) which serves a http server + webinterface. When the user uses the save functionality a json configuration file is written to a predefined location on the TV where the new PicCap service can pick it up and work with it. 
This would be the MVP in my eyes. After that it could be extended for more features like multiple configurations (if needed), screen live view or whatever is needed.

**Comment by psychowood** [comment]
Which settings do you expect to change that frequently to the point of needing a web interface?
I mean, it'd be useful, sure, but not really an MVP imho, considering the set and forget nature of this app...

Just my 2 cents, of course :)

**Comment by nnmdd** [comment]
at first look web interface can be used to check app startup and if it not - start with dev tools/ssh for example

**Comment by LegendaryB** [comment]
@psychowood: You are right about the fact that its mostly a config and forget thing. But you could also integrate features which make it more comfortable to use for users who don't have so much technical knowledge. For example we could show all the log messages in a text area or a button to download the log. In case of an error a user could simply copy the output and pass it directly to discord and/or a github issue etc.

**Comment by Cellivar** [comment]
As someone who doesn't own the magic remote and is stuck using buttons, being able to configure the settings from my phone would be so much nicer.
